README.md

```diff
@@ -63,15 +63,59 @@ Then add `__tests__` to `sources` in your `bsconfig.json`:
 ```js
 "sources": [
   {
-    "dir": "src"
+    "dir": "app",
+    "subdirs": true
   },
   {
     "dir": "__tests__",
-    "type": "dev"
+    "type": "dev",
+    "subdirs": true
   }
 ]
 ```
 
+Then your project structure would look like this:
+
+```
+src/
+├── MyComponent.re
+__tests__/
+├── MyComponent_test.re
+README.md
+
+
+(NOTE: no .re files in the root of src, if you put them there they will not be built)
```

Comment:
This also seems unnecessary here as it's unrelated to the workings of bs-jest. It's good advice, but it should be included in the bucklescript docs instead.